### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by hmgaudecker**
- Remove unnecessary `__init__.py` - files from screencast directories.

### CODE REVIEW

File: src/epp_topics/python_basics/dicts/objectives_materials.ipynb [hunk 1]
```diff
@@ -79,6 +79,73 @@
...
     "    {\n",
     "        \"question\": (\n",
+    "            \"Which of the following types of objects can you use as dictionary keys?\"\n",
+    "        ),\n",
+    "        \"type\": \"many_choice\",\n",
+    "        \"answers\": [\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A list.\"\"\",\n",
+    "                \"correct\": False,\n",
+    "                \"feedback\": \"Correct!\",\n",
```
Comment (janosg) [inline src/epp_topics/python_basics/dicts/objectives_materials.ipynb:89]: Feedback should be "Incorrect"

File: src/epp_topics/python_basics/dicts/objectives_materials.ipynb [hunk 2]
```diff
@@ -79,6 +79,73 @@
...
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A tuple.\"\"\",\n",
+    "                \"correct\": True,\n",
+    "                \"feedback\": \"Incorrect.\",\n",
```
Comment (janosg) [inline src/epp_topics/python_basics/dicts/objectives_materials.ipynb:94]: Feedback should be correct

File: src/epp_topics/python_basics/dicts/objectives_materials.ipynb [hunk 3]
```diff
@@ -79,6 +79,73 @@
...
+    "                \"feedback\": \"\"\"Incorrect.\"\"\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"A floating point number.\"\"\",\n",
+    "                \"correct\": True,\n",
+    "                \"feedback\": \"\"\"Correct, though not recommended due to floating point\n",
+    "                    precision issues.\"\"\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"answer\": \"\"\"An Boolean.\"\"\",\n",
```
Comment (janosg) [inline src/epp_topics/python_basics/dicts/objectives_materials.ipynb:113]: A boolean, not An boolean

**Review by janosg — APPROVED** [review]
Just small typos, I already approve.

File: src/epp_topics/python_basics/dicts/objectives_materials.ipynb (hunk 3 above)
Comment (hmgaudecker) [reply src/epp_topics/python_basics/dicts/objectives_materials.ipynb:113]: https://en.wikipedia.org/wiki/George_Boole

**Pull request merged**